.github/workflows/ecs-deployment.yml

```diff
@@ -78,9 +78,6 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
           build-args: |
-            NEXTAUTH_SECRET=${{ env.NEXTAUTH_SECRET }}
-            DATABASE_URL=${{ env.DATABASE_URL }}
-            ENCRYPTION_KEY=${{ env.ENCRYPTION_KEY }}
```

Comment:
<img width="633" alt="Screenshot 2024-03-14 at 09 59 08" src="https://github.com/formbricks/formbricks/assets/675065/36dba80f-09c0-4e6a-9710-2ea90b457fd7">

We also generate them in the workflow files. Can you please remove those commands, too :-)